recipes/enaml/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.10.1" %}
+package:
+  name: enaml
+  version: {{ version }}
+
+source:
+  url: https://github.com/nucleic/enaml/archive/{{ version }}.tar.gz
+  fn: enaml-{{ version }}.tar.gz
+  sha256: f6500bda40cdff006b4dc0ea7f4a0ae820542b2fbae3362f48dcc2194d84dfcc
+
+build:
+  number: 0
+  entry_points:
+    - enaml-run = enaml.runner:main
+  osx_is_app: True
+  skip_compile_pyc:
```

Comment:
Why is the compile skip necessary? It may very well be, I just haven't seen it in other recipes.

Reply:
Enaml includes some files that are used only on specific version of Python and rather than removing them using a patch I prefer to simply skip the compilation. This way the installed package looks more like the full source.